app/components/interview_bookings_item_component.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if in_the_past? %>
+  <p class='govuk-body'>You had an interview on <%= formatted_date(interview) %></p>
+<% else %>
+  <p class='govuk-body'>You have an interview scheduled for <%= formatted_time(interview) %></p>
+  <p class="govuk-body govuk-!-margin-bottom-0">Information from provider:</p>
+  <div class="govuk-inset-text govuk-!-margin-top-1 govuk-!-margin-bottom-1">
+    <%= location(interview) %>
+    <%= additional_details(interview) %>
+  </div>
```

Comment:
We have a component for Inset text we can use, I think?

Reply:
Yes, just tried but the component doesn't take a block, just a `text` arg. So I think I'll have to leave as is for now. https://github.com/DFE-Digital/govuk-components/blob/master/app/components/govuk_component/inset_text.rb

Reply:
It’s on it’s way! https://github.com/DFE-Digital/govuk-components/issues/134